app/store/migrations/migrations.js

```diff
@@ -0,0 +1,13 @@
+// Intentionally in JS to allow ensuring we can directly edit state between migrations.
```

Comment:
not sure you need this comment, as itd be a bit odd to expect to also version the typesystem. That said, not sure its related to being able to mutate the state? ts doesnt preclude you from mutations?